doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -106,6 +106,11 @@ To submit the Ray Job, we use ``ray job submit``:
 
 By default, this command will run the script on the Ray Cluster and wait until the job has finished. Also note the ``hello world`` printout: the stdout of the job is streamed back to the client.
 
+.. note::
```

Comment:
Could we actually just use this command in the example? I think that will be the least confusing, so there aren't two different versions of the command.

Also, can you include a brief explanation of what the argument does under the hood?

Reply:
Done, LMK what you think.